pkg/rule/rule.go

```diff
@@ -191,3 +198,14 @@ func removeInlineIgnore(line string) string {
 func (r *Rule) Disabled() bool {
 	return len(r.Terms) == 0
 }
+
+// SetAddNoteToMessage populates AddNoteToMessage attributte in Options
+// Options.AddNoteToMessage is ussed in ReasonWithNote
+// If "add_note_to_message" is already defined for the rule in yaml, it will not be overridden
+func (r *Rule) SetAddNoteToMessage(addNoteToMessage bool) {
+	if r.Options.AddNoteToMessage != nil {
+		return
```

Comment:
Can you add a test case for this scenario?

Reply:
It is covered [here](https://github.com/eedorenko/woke/blob/9d8f8fe94b5b309a4a7af0fc0c7a8126217d36f5/pkg/config/config_test.go#L120) and [here](https://github.com/eedorenko/woke/blob/9d8f8fe94b5b309a4a7af0fc0c7a8126217d36f5/pkg/config/config_test.go#L135)